test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // no null objects
+        assertNoNull("");
+        assertNoNull(2, "hello", "");
+        assertNoNull("null");
+
+        // have null objects
+        assertAnyNull();
```

Comment:
Please note that an empty list is not equivalent to null. `Util.isAnyNull()` is expected to return `false`.